README.md

```diff
@@ -67,14 +67,20 @@ Start by forking the repository https://github.com/NationalGenomicsInfrastructur
 
 If you want to test your roles/playbook run `ansible-playbook install.yml`. This will install your development run in `/lupus/ngi/irma3/devel-root/<username>_<branch_name>`.
 
-When you are satisfied with your changes you need to test it in staging. First create a pull request from your feature branch into upstream irma-provisioning's master branch.
+When you are satisfied with your changes you need to test it in staging. To do this, you must create a pull request to one of the two staging branches of irma-provision.
 
-Do the following once the feature has been approved:
+There are two staging branches in the repository, one called monthly and the other bimonthly. The changes that do not need extensive testing and validation on stage should be pushed to the monthly branch, which is used to make stage and production deployments on a monthly cycle. Changes that need time for validation on stage (like pipeline version updates) should be pushed to the bimonthly branch.
+
+Production deployments on the monthly release cycle are done on the last Monday of each month. A stage deployment from the monthly branch will be made two weeks before this date and the introduction of new changes to the monthly branch will be frozen. Only fixes to changes already merged will be approved to the monthly branch during this time.
+
+Stage deployments from the bimonthly branch can be made outside of the monthly release cycle. The changes on the bimonthly branch would generally be pulled to the master branch and deployed to production in the monthly release cycle once all validations are complete.
```

Comment:
I wonder if it's worth mentioning that a PR should be made to the master branch.

Maybe it would make sense to first merge bimonthly with monthly and then merge monthly with master?

Reply:
Hmm, yeah maybe that works better.